week-1/I-floats/exercise.js

```diff
@@ -1,2 +1,6 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+var percentstudent = Math.floor(numberOfStudents);
```

Comment:
nice codes, you can also use Math.round() instead of Math.floor().

Reply:
@laperla11 thanks for the comment, you are right, will implement the change.